osu.Game/Overlays/BeatmapSet/BeatmapSetHasVideoBadge.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Cursor;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Localisation;
+using osu.Game.Resources.Localisation.Web;
+using osuTK;
+using osuTK.Graphics;
+
+namespace osu.Game.Overlays.BeatmapSet
+{
+    public partial class BeatmapSetHasVideoBadge : CircularContainer, IHasTooltip
```

Comment:
`VideoIconPill` exists already for beatmap cards in listing overlay, any reason why it's not used instead of this?

Reply:
Wasn't aware that's a thing. I've changed it to `VideoIconPill` and added a storyboard icon as well